config/config.yaml

```diff
@@ -101,6 +102,7 @@ datasets:
     -
       # Labels can only contain letters, numbers, or underscores
       label: data0
+      # To run OmicsIntegrator1 with dummy nodes, add the dummy.txt file to node_files
       node_files: ["node-prizes.txt", "sources.txt", "targets.txt"]
```

Comment:
Can you add the dummy.txt file to the list of node_files for both data0 and data1

Reply:
Is this an intended behavior: If I have the node_prizes file with an dummy column but then I add a non empty dummy.txt file to the list of node_files, should there be a prepared file for oi1 that is non-empty or should it listen to what the empty dummy column is "saying" and there should be a prepared file for oi1 that is empty?

Reply:
When I do the behavior above locally, I get back an empty dummy nodes files in prepared inputs for oi1 for both data0 and data1

Reply:
Let's not focus too much on the behavior when we have dummy column in the node-prizes file and a dummy.txt file. I opened https://github.com/Reed-CompBio/spras/issues/192 to propose dropping the seconary file format.

We found the data loader keeps existing columns (node attributes) and drops new duplicates:
https://github.com/Reed-CompBio/spras/blob/a26f4d0ea86e4bf8e3fb89fcca5c33e44103d7c5/spras/dataset.py#L120-L123